streamer/src/nonblocking/quic.rs

```diff
@@ -130,17 +130,28 @@ pub fn spawn_server(
     max_streams_per_ms: u64,
     wait_for_chunk_timeout: Duration,
     coalesce: Duration,
-) -> Result<(Endpoint, Arc<StreamStats>, JoinHandle<()>), QuicServerError> {
+) -> Result<
+    (
+        Endpoint,
+        Arc<StreamStats>,
+        JoinHandle<()>,
+        u32, /* max_concurrent_connections */
+    ),
+    QuicServerError,
+> {
     info!("Start {name} quic server on {sock:?}");
-    let (config, _cert) = configure_server(keypair)?;
+    let concurrent_connections = max_staked_connections + max_unstaked_connections;
+    let max_concurrent_connections = (concurrent_connections + concurrent_connections / 4) as u32;
```

Comment:
This mixing `u32` and `usize` for `max_..._connections` isn't good.
Ideally they should all have the same type.

Reply:
The api we are using is u32, I will see why we picked usize in the beginning. maybe because of the connection table size is usize?